supernode/daemon/mgr/task/manager_util.go

```diff
@@ -116,36 +118,33 @@ func (tm *Manager) updateTask(taskID string, updateTaskInfo *types.TaskInfo) err
 		return err
 	}
 
-	if !isSuccessCDN(updateTaskInfo.CdnStatus) {
-		// when the origin CDNStatus equals success, do not update it to unsuccessful
-		if isSuccessCDN(task.CdnStatus) {
-			return nil
-		}
-
-		// only update the task CdnStatus when the new CDNStatus and
-		// the origin CDNStatus both not equals success
+	// Update the origin task CDNStatus when the origin CDNStatus not equals success.
+	if !isSuccessCDN(task.CdnStatus) {
```

Comment:
The logic here has been changed. When the `update task` succeeds, the `origin task` should be updated regardless of whether it was successful before.

Reply:
Yes, thanks for pointing out this! I will change it.